.github/workflows/validate-semantic-pr.yml

```diff
@@ -31,6 +31,10 @@ jobs:
             revert
             style
             test
+          scopes: |
+            scheduler
+            operator
+            functions-runtime
```

Comment:
would add additional scopes for doc and examples ...

Reply:
documentation and examples should be covered by the docs semantic commit type i'd say..
so you could do something like `docs: Add new docs page for XY`